### ISSUE TITLE
The "Find a site near you" link isn't actionable to keyboard users with or without TalkBack running [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#237]
On the **keep-Safe** screen, the "Get Tested" section contains a link to "Find a site near you", which is announced, along with the rest of the text, but TalkBack users cannot double tap to active the link. The link's role isn't conveyed to TalkBack users as well.

![keep-Safe - Link - Find a site near you TB running](https://user-images.githubusercontent.com/7034772/92021412-b7b7b980-ed27-11ea-820f-725d6a1d19d6.png)

**Comment by floridemai** [comment]
@jonnyjames Can you check in the next build please. Thanks

**Comment by jonnyjames** [comment]
@floridemai This issue is similiar to #248, but in this case, using _**only**_ the keyboard (as the other issue covers the keyboard + TalkBack issue) of the "Find a site near you" link, not receiving focus, and is not actionable using the keyboard alone.

**Comment by floridemai** [comment]
@jonnyjames Can we close this one out please as it fixed the original issue and open a separate one (maybe a generic one) for keyboard (which is something we were not aware nor worked on). Thanks

**Comment by AlanSl** [comment]
I believe the external keyboard support will be in the next build as https://github.com/project-vagabond/covid-green-app/pull/399 was merged overnight

**Comment by jonnyjames** [comment]
@AlanSl The issue with not being able to reach the two links persists on those two screens.
